harness/determined/launch/autohorovod.py

```diff
@@ -81,6 +58,9 @@ def main() -> int:
     # easiest way to get the chief_ip is with container_addrs.
     chief_ip = info.container_addrs[0]
 
+    # Chief IP is set as an environment variable to support multiple launch layers
```

Comment:
nit: can you say "to support nested launch layers"?  Because supporting "multiple launch layers" could also mean autohorovod vs torch-native vs tensorflow-native launching, or something like that.